icu4c/source/common/punycode.cpp

```diff
@@ -385,6 +390,12 @@ u_strFromPunycode(const UChar *src, int32_t srcLength,
     if(srcLength==-1) {
         srcLength=u_strlen(src);
     }
+    if (srcLength > 2000) {
```

Comment:
Declare this 2000 as a const with a meaningful name for it.

Reply:
Seems unnecessary since we use it exactly once, and it has a comment.

Reply:
now done